subprojects/docs/src/docs/userguide/kotlin_dsl.adoc

```diff
@@ -410,19 +428,56 @@ Favor <<multi_project_builds#sec:decoupled_projects,decoupling your projects>> i
 // TODO note that these will be deprecated soon, for Gradle 5?, need an issue link
 --
 
+Multi-project builds are very common.
+Android builds are, for example, multi-project builds from the start.
+When creating a new build from Android Studio it creates a multi-project build.
+
+To get the most benefits of the Gradle Kotlin DSL you should strive to keep your build scripts declarative.
+The main thing to remember here is that in order to get type-safe accessors, plugins must be applied before the body of build scripts.
+As seen above, this can be achieved by using the `plugins {}` block.
+
+[[sec:multi_project_builds_applying_plugins]]
+=== Applying plugins
+
+In multi-project builds you should centrally declare plugin dependencies in the root project build file, specifying the version, without applying those used in sub-projects only.
+This always allow for applying the plugins in sub projects build scripts using the `plugins {}` block.
+
+.Declare plugin dependencies in the root build script using the `plugins {}` block
+====
+include::sample[dir="kotlinDsl/multiProjectBuild",files="settings.gradle.kts[tags=base];build.gradle.kts[tags=root];domain/build.gradle.kts[];infra/build.gradle.kts[];http/build.gradle.kts[]"]
+====
+
 If your build requires additional plugin repositories, they should be declared in the `pluginManagement {}` block in your `settings.gradle.kts` file:
 
 .Declare additional plugin repositories
 ====
 include::sample[dir="kotlinDsl/multiProjectBuild",files="settings.gradle.kts[tags=repositories]"]
 ====
 
+Plugins fetched from a source other than the link:https://plugins.gradle.org/[Gradle Plugin Portal] may or may not be usable with the `plugins {}` block.
+It depends on how they have been published.
+
+For example, the Android Plugin for Gradle is not published to the Gradle Plugin Portal and at least up to version 3.2.0 of the plugin, the metadata required to resolve the artifacts for a given plugin identifier is not published to the Google repository. Please star link:https://issuetracker.google.com/issues/64551265[the corresponding Google issue].
+
+In that case, you can fall back to declaring your plugin dependencies using the `buildscript {}` block in the root project build script:
```

Comment:
This isn't entirely accurate, wouldn't it be better to discuss using `pluginManagement` in the settings script?

Reply:
I suppose you do do this in the next paragraph. I guess it's just kind of confusing that you say "may not be usable with the `plugins {}` block". Aren't all plugins usable with the plugins block as long as you use `pluginManagement`.

The only case where I know `plugins {}` is truly unusable is when your plugin comes from an included build.

Reply:
If you don't *need* to use the `plugins {}` block in your root project build script, falling back to `buildscript {}` makes sense, and probably very similar to what many users already do, like Android folks. The goal of explaining both strategies is not to disturb people habits too much, the motivation comes from feedback from several sources.

You're right about the phrasing though, I'll refine.

BTW, with the right `pluginManagement` configuration you can use plugins from included builds, see this WIP sample https://github.com/gradle/kotlin-dsl/blob/eskatos/samples/composite-build-logic/samples/composite-build-logic/build.gradle.kts#L4